subliminal/providers/podnapisi.py

```diff
@@ -116,20 +137,30 @@ def query(self, language, series=None, season=None, episode=None, title=None, ye
         logger.debug('Searching episode %r', params)
         subtitles = []
         while True:
-            root = self.get('/search', params)
+            root = self.get('/ppodnapisi/search', params)
             if not int(root.find('pagination/results').text):
                 logger.debug('No subtitle found')
                 break
             if series and season and episode:
-                subtitles.extend([PodnapisiSubtitle(language, int(s.find('id').text), s.find('release').text.split(),
+                try:
+                    subtitles.extend([PodnapisiSubtitle(language, int(s.find('id').text), s.find('release').text.split(),
                                                     'h' in (s.find('flags').text or ''), s.find('url').text[38:],
                                                     series=series, season=season, episode=episode)
                                   for s in root.findall('subtitle')])
+                except AttributeError:
+                    # there simply wasn't enough information in the TV Show
+                    # gracefully handle this instead of crashing :)
+                    break
```

Comment:
Let me see what i can do about this.  There should be no reason i can't accommodate both of your very valid points.